schematools/importer/base.py

```diff
@@ -446,7 +468,16 @@ def table_factory(
             col_kwargs["nullable"] = False
             col_kwargs["autoincrement"] = False
 
-        id_postfix = "_id" if field.relation else ""
+        id_postfix = ""
+        if field.relation:
+            id_postfix = "_id"
+            if relname_from_identifier:
+                dataset_id, rel_table_id = field.relation.split(":")
```

Comment:
Wellicht hier nog wat extra precaution toevoegen. De tabelnaam in de relatie kan evt. in camelCase gedefinieerd zijn, maar zal uiteindelijk in snake_case in de kolomdefinitie terecht moeten komen.

Reply:
Zat daar even mee te stoeien. De `to_snake_case` staan her en der verspreid. Liefst zou ik zien dat als het schema is ingelezen alles wat een kolom- of tabelnaam kan zijn in één klap naar snake case wordt geconverteerd. Dan hoef je er later, dieper in de code, niet meer over na te denken. Soort van transformatie stap (in pseudo code):

```python
read_schema >> transform_schema >> generate_tables >> generate_indices
```

`transform_schema` zou dan alle naamgeving (kolom- en tabelnamen) moeten omzetten naar snake case. De latere stappen hoeven dan niet meer na te denken of ze met camelCase danwel snake case te maken hebben.

Reply:
Maar dat is een iets te grote refactor op dit moment. Zal de precaution toevoegen.